src/components/SlateEditor/plugins/related/EditRelated.tsx

```diff
@@ -0,0 +1,247 @@
+/**
+ * Copyright (c) 2017-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import React, { useState } from 'react';
+import { useTranslation, withTranslation } from 'react-i18next';
+import Button from '@ndla/button';
+import styled from '@emotion/styled';
+import darken from 'polished/lib/color/darken';
+import { DragDropContext, Droppable, Draggable } from 'react-beautiful-dnd';
+import Tooltip from '@ndla/tooltip';
+import { Pencil } from '@ndla/icons/action';
+import { colors, spacing } from '@ndla/core';
+import { search } from '../../../../modules/search/searchApi';
+import AsyncDropdown from '../../../Dropdown/asyncDropdown/AsyncDropdown';
+import Overlay from '../../../Overlay';
+import RelatedArticle from './RelatedArticle';
+import ContentLink from '../../../../containers/ArticlePage/components/ContentLink';
+import DeleteButton from '../../../DeleteButton';
+import { RelatedArticleType } from './RelatedArticleBox';
+
+const StyledContainer = styled('div')`
+  position: absolute;
+  width: 100%;
+`;
+
+const StyledBorderDiv = styled('div')`
+  position: relative;
+  border: 2px solid ${colors.brand.tertiary};
+  padding: ${spacing.large};
+  padding-top: 0;
+  background-color: ${colors.white};
+  z-index: 1;
+`;
+
+const StyledListWrapper = styled('div')`
+  display: flex;
+  flex-wrap: wrap;
+  z-index: 1;
+
+  & figure {
+    position: static !important;
+  }
+`;
+
+const StyledArticle = styled('div')`
+  position: relative;
+  flex: 1 0 50%;
+  max-width: 600px;
+
+  & > article {
+    max-width: 100%;
+  }
+`;
+
+const StyledOr = styled('div')`
+  margin: 10px 0;
+`;
+
+const StyledEditButton = styled(Button)`
+  position: absolute;
+  top: 0.1rem;
+  right: 1.5rem;
+  color: ${colors.support.red};
+
+  &:hover,
+  &:focus {
+    color: ${darken(0.2, colors.support.red)};
+  }
+`;
+
+const StyledDropZone = styled('div')`
+  flex: 1;
+`;
+
+interface Props {
+  onRemoveClick: (e: Event) => void;
+  updateArticles: (newArticles: RelatedArticleType[]) => void;
+  onExit: () => void;
+  articles: RelatedArticleType[];
+  onInsertBlock: (newArticle: string) => void;
+  insertExternal: (url: string, title: string) => Promise<void>;
+}
+
+const EditRelated = ({
+  onRemoveClick,
+  updateArticles,
+  insertExternal,
+  articles,
+  onInsertBlock,
+  onExit,
+  ...rest
+}: Props) => {
+  const { t } = useTranslation();
```

Comment:
Hvis du skal bruke `useTranslation` er det ikke behov for å wrappe komponenten i `withTranslation`.